src/azure-cli/azure/cli/command_modules/resource/tests/latest/management_group_level_parameters.json

```diff
@@ -2,14 +2,8 @@
   "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
   "contentVersion": "1.0.0.0",
   "parameters": {
-    "targetMG": {
-      "value": "cli_test_management_group_deployment"
-    },
     "nestedsubId": {
-      "value": "fb3a3d6b-44c8-44f5-88c9-b20917c9b96b"
-    },
-    "nestedRG": {
-      "value": "cli_test_management_group_deployment"
```

Comment:
May I know why we remove such parameter values?

Reply:
As above comment, some data is no longer needed. Some other data is not written in the configuration parameters, changed to get in the code.